packages/brand-moment/src/BrandMoment.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode, ReactElement } from "react"
+import { Heading, Box, Paragraph } from "@kaizen/component-library"
+import { Button, ButtonProps } from "@kaizen/draft-button"
+import { SceneProps } from "@kaizen/draft-illustration"
+import { assetUrl } from "@kaizen/hosted-assets"
+import classnames from "classnames"
+import styles from "./BrandMoment.scss"
+
+type Props = {
+  backgroundColor: "blue" | "purple" | "green" | "red"
+  header: ReactNode
+  illustration: ReactElement<SceneProps>
+  heading: ReactNode
+  subheading?: ReactNode
+  body?: ReactNode
+  primaryAction: ButtonProps
+  secondaryAction?: ButtonProps
+  footer?: ReactNode
```

Comment:
We discussed if the footer should contain the full footer as used in capture, or if it should be passed in, similar to the nav header.

For now, the only use case for the footer is capture.

We're intending to leave it (have the component define the footer layout, and the props pass in the footer content). This makes it easier to consume, and doesn't give flexibility we don't need.

One suggestion is to rename the prop to `footerContent` so that if we change our minds in future, we can enforce the change with the type system by creating a new prop (and perhaps removing the old prop if desired):

```suggestion
  footerContent?: ReactNode
```